resources/schema/mutations.py

```diff
@@ -466,7 +466,6 @@ class PierInput(AbstractAreaSectionInput):
 class CreatePierMutation(graphene.ClientIDMutation):
     class Input(PierInput):
         harbor_id = graphene.ID(required=True)
-        price_tier = graphene.Field("payments.schema.PriceTierEnum", required=True)
```

Comment:
Should this instead be kept and only the `required` part removed?

Reply:
The `price_tier` is already defined on the `PierInput` as non-required.

Reply:
👍